test.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'config/environment'
```

Comment:
🤔

Reply:
This is commited my mistake - will remove it.